Aptoide/aptoide-client-v8
# [APP-2413] - Add editorial screen, home sections and see all

**What does this PR do?**

This PR adds the remaining Home Sections bundle views, as well as the Editorial Screen and SeeAll views that were missing.

Some screens will return an error view due to having catappult apps there which breaks the listing on our side, I believe.

Editors' choice doesn't have a See All because it's bundle tag doesn't end in "more" due to some odd error in the getStore Widgets.

Install buttons on home still do not work and a new ticket will be created regarding this issue.

All of this was implemented with the GH design, so new colors and typographies are temporary. Aptoide Games is for future tickets.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] CategoriesView.kt
- [ ] EditorialView.kt
- [ ] EditorialViewScreen.kt
- [ ] AppGridView.kt
- [ ] CarouselAppView.kt
- [ ] CarouselLargeAppView.kt
- [ ] MoreBundleView.kt
- [ ] MyGamesBundleView.kt
- [ ] PublisherTakeOverBundle.kt
- [ ] SeeAllMyGamesScreen.kt
- [ ] SeeAllMyGamesUiState.kt
- [ ] SeeAllMyGamesViewModel.kt
- [ ] SearchView.kt
- [ ] AppColors.kt
- [ ] AppTypography.kt
- [ ] strings.xml

**How should this be manually tested?**

  Test all the new views (if possible) to see if they are OK. 
  Flow on how to test this or QA Tickets related to this use-case: [APP-2413](https://aptoide.atlassian.net/browse/APP-2413)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2413](https://aptoide.atlassian.net/browse/APP-2413)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 



**Code Review Checklist**


[APP-2413]: https://aptoide.atlassian.net/browse/APP-2413?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
